karate-tests/src/test/java/mocks/sandbox/patch-patient.js

```diff
@@ -141,6 +141,16 @@ function patchPatient (originalPatient, request) {
         updatedPatient.name.splice(1, 1)
       }
     }
+
+    if (patch.op === 'remove' && patch.path === '/name/0') {
+      if (request.body.patches[0].value.use === 'usual') {
```

Comment:
Does this want to be?
```suggestion
      if (patch.value.use === 'usual') {
```